src/components/Nav/CaseMenu.jsx

```diff
@@ -5,18 +5,31 @@ import { Panel } from 'nav-frontend-paneler';
 import { Normaltekst } from 'nav-frontend-typografi';
 import { EtikettInfo } from 'nav-frontend-etiketter';
 import './CaseMenu.less';
+import { toDate } from '../../utils/date';
 
-const CaseMenu = ({ behandling }) => {
-    const { behandlingsId } = behandling;
+const CaseMenu = ({ behandlinger, behandling, setValgtBehandling }) => {
     const { arbeidsgiver, fom, tom } = behandling.originalSøknad;
     const { sykmeldingsgrad } = behandling.periode;
+    const behandlingMapper = behandling => ({
+        behandlingsId: behandling.behandlingsId,
+        fom: behandling.originalSøknad.fom,
+        tom: behandling.originalSøknad.tom
+    });
+    const items = behandlinger.map(behandling => behandlingMapper(behandling));
+    const currentItem = behandlingMapper(behandling);
+    const itemLabel = item => `${toDate(item.fom)} - ${toDate(item.tom)}`;
```

Comment:
Er det overkill med all denne mappingen, kan man ikke bare bruke behandling.xxx direkte? Kjør debatt!

Reply:
Formålet er kanskje å gjøre Picker-komponenten generisk. Så kan man nok argumentere for at det er litt prematurt når den bare brukes ett sted..
Kanskje det hadde føltes hakket bedre å gjøre mappingen i `CasePicker` i stedet, da skjer det litt nærmere der det trengs. Kan også se på navngivingen, `items` fungerer bra inne i `Picker`, men her ute i `CaseMenu` hadde det kanskje gjort seg med et mer spesifikt navn. Men navngiving er jo vanskelig.

Reply:
Var for å beholde den som generisk, ja. Men nå fjernet jeg hele CasePicker, den ble egentlig helt unødvendig.